homeassistant/components/mqtt/debug_info.py

```diff
@@ -5,34 +5,41 @@
 from collections.abc import Callable
 import datetime as dt
 from functools import wraps
-from typing import Any
+from typing import TYPE_CHECKING, Any
 
 import attr
 
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import entity_registry as er
 from homeassistant.util import dt as dt_util
 
-from .const import ATTR_DISCOVERY_PAYLOAD, ATTR_DISCOVERY_TOPIC
+from .const import ATTR_DISCOVERY_PAYLOAD, ATTR_DISCOVERY_TOPIC, DATA_MQTT
 from .models import MessageCallbackType, PublishPayloadType
 
-DATA_MQTT_DEBUG_INFO = "mqtt_debug_info"
+if TYPE_CHECKING:
+    from .mixins import MqttData
+
+
```

Comment:
This becomes unnecessary if #78825 is implemented.

Reply:
Nice. I'll have a look.

Reply:
Okay, I missed that one. I suggest we merge #78825 first, then I'll correct / rebase my PR's.